src/rebase.c

```diff
@@ -1327,6 +1327,34 @@ int git_rebase_finish(
 	return error;
 }
 
+const git_oid *git_rebase_orig_head_id(git_rebase *rebase) {
+	if (rebase->type == GIT_REBASE_TYPE_MERGE) {
+		return &rebase->orig_head_id;
+	}
+	return NULL;
+}
```

Comment:
Nit: we don't do braces around single lines.  eg:

```
if (rebase->type == GIT_REBASE_TYPE_MERGE)
    return &rebase->orig_head_id;

return NULL;
```

But tbqh, I would just:

```
return (rebase->type == GIT_REBASE_TYPE_MERGE) ? &rebase->orig_head_id : NULL;
```

and for the strings you can just return them directly, no?  (They're either null or they're not, no need to check the rebase type).

But that's less important

Reply:
I had it that way before but I was not sure if that would remain the same for other rebase types, since other rebase types can have different file structures/objects inside the `.git/rebase-merge` dir (i'm currently looking into implementing interactive rebase).

But I guess for now thats true, and it probably wont make a difference if I return it directly.